api/envoy/config/core/v3/base.proto

```diff
@@ -430,9 +430,26 @@ message DataSource {
 }
 
 // The message specifies the retry policy of remote data source when fetching fails.
+// [#next-free-field: 7]
 message RetryPolicy {
   option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.core.RetryPolicy";
 
+  message RetryPriority {
```

Comment:
To the extent these are repeated entries of what we have in router::RetryPolicy, I think it would be good to clearly document the relationship.

Reply:
Ah, added. 

in addition to per field commetn, I hope the parent message udpa annotation could give reader a hint what this message is meant for.